### PR TITLE
Code coverage integration and add migrations to git repository

### PR DESCRIPTION
**Pull request by iroshni**
**Fixed Issues**
code coverage and appveyor setup, removed migration folder from gitignore to apply migrations at run time.

**Files Changed**
DBUtility.cs - removed code to delete database (as migrations we not applied.)
Startup.cs - removed code to delete database (as migrations we not applied.)
appveyor.yml - added to setup appveyor
migrations files - added initial migrations.

**Checks**

- [x] Naming Conventions
- [x] Server side code comments (proper English, grammar and no spelling mistakes)
- [ ] Client side code comments (proper English, grammar and no spelling mistakes)
- [x] Unused variables, methods, blank spaces and name spaces. 
- [x] Optimal Code and code formatting
- [x] Commit History is proper, no merge is done. 
- [x] Commit/pull request messages should be proper to justify your feature
- [x] All test cases are executed provided by the tester.

**Known Issues**

**Comments**

### CODE REVIEW

File: Trappist/src/Promact.Trappist.Web/Migrations/20170410063542_initial.cs [hunk 1]
```diff
@@ -0,0 +1,459 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Promact.Trappist.Web.Migrations
+{
+    public partial class initial : Migration
```
Comment (chintans) [inline Trappist/src/Promact.Trappist.Web/Migrations/20170410063542_initial.cs]: MINOR Rename class "initial" to match camel case naming rules, consider using "Initial". [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS101)

**Comment by chintans** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DbUtility.cs#L17](https://github.com/Promact/trappist/blob/cb5fa33982889a13618edf6fbd1f2713df8db6cc/Trappist/src/Promact.Trappist.Utility/DbUtil/DbUtility.cs#L17): Remove the "_hostingEnv" field and declare it as a local variable in the relevant methods. [rule](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by chintans** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by iroshni** [comment]
I need to integrate coveralls and check issues with missing pdbs.

**Pull request merged**